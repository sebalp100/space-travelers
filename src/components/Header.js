import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../planet.png';

function Header() {
  return (
    <nav>
      <div className="navDiv">
        <img alt="icon" src={logo} height="50" />
        <h1>Space TravelersÂ´ Hub</h1>
      </div>
      <div className="link">
        <li>
          <NavLink
            to="/"
            className="navLink"
          >
            Rockets
          </NavLink>
        </li>
        <li>
          <NavLink
            to="/Missions"
            className="navLink"
          >
            Missions
          </NavLink>
        </li>
        <li>
          <NavLink
            to="/Dragons"
            className="navLink"
          >
            Dragons
          </NavLink>
        </li>
        <li>
          <NavLink
            to="/Profile"
            className="navLink"
          >
            My Profile
          </NavLink>
        </li>
      </div>
    </nav>
  );
}

export default Header;
