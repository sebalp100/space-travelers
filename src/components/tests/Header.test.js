import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Header from '../Header';

describe('Header', () => { });
it('Testing header component rendering', () => {
  const component = render(
    <BrowserRouter>
      <Header />
    </BrowserRouter>,
  );
  expect(component).toMatchSnapshot();
});
