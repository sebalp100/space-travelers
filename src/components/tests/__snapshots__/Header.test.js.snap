// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing header component rendering 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <div
          class="navDiv"
        >
          <img
            alt="icon"
            height="50"
            src="planet.png"
          />
          <h1>
            Space Travelers´ Hub
          </h1>
        </div>
        <div
          class="link"
        >
          <li>
            <a
              aria-current="page"
              class="navLink active"
              href="/"
            >
              Rockets
            </a>
          </li>
          <li>
            <a
              class="navLink"
              href="/Missions"
            >
              Missions
            </a>
          </li>
          <li>
            <a
              class="navLink"
              href="/Dragons"
            >
              Dragons
            </a>
          </li>
          <li>
            <a
              class="navLink"
              href="/Profile"
            >
              My Profile
            </a>
          </li>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav>
      <div
        class="navDiv"
      >
        <img
          alt="icon"
          height="50"
          src="planet.png"
        />
        <h1>
          Space Travelers´ Hub
        </h1>
      </div>
      <div
        class="link"
      >
        <li>
          <a
            aria-current="page"
            class="navLink active"
            href="/"
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            class="navLink"
            href="/Missions"
          >
            Missions
          </a>
        </li>
        <li>
          <a
            class="navLink"
            href="/Dragons"
          >
            Dragons
          </a>
        </li>
        <li>
          <a
            class="navLink"
            href="/Profile"
          >
            My Profile
          </a>
        </li>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
